index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        h1 {text-align: center;
+        color: rgb(228, 56, 56);
+        background-color: thistle;
+        }
+        h3 {background-color: rgb(9, 212, 185);}
```

Comment:
styles for h3 should be in style.css and add the class to h3 instead of styling by tag